docs/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/jefBinomed/SfeirSchoolTheme.git"
+    "url": "git+https://github.com/sfeir-open-source/sfeir-school-theme.git"
```

Comment:
Are you sure you don't want to reference our React school repo instead ?

Reply:
sure :) done!